vaccine_feed_ingest/runners/il/sfsites/normalize.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import pathlib
+import re
+import sys
+from typing import List, Optional
+
+import pydantic
+from vaccine_feed_ingest_schema import location as schema
+
+from vaccine_feed_ingest.utils.log import getLogger
+
+logger = getLogger(__file__)
+
+URL_RE = re.compile(
+    r"(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})"
+)
```

Comment:
i can drop this in `utils/normalize.py` if that's helpful

Reply:
That seems useful (in a followup) - we have a few regexes like this floating around.